src/main/java/teammates/ui/controller/InstructorCourseAddAction.java

```diff
@@ -109,10 +106,6 @@ private void createCourse(CourseAttributes course) {
         } catch (InvalidParametersException e) {
             setStatusForException(e);
         }
-
-        if (isError) {
-            return;
-        }
```

Comment:
Unrelated change? Seems like `isError` is never set to true anywhere, but was never set to true even in the previous code, so this is unrelated.

Reply:
It's set to true in `setStatusForException`. But anyway, it's unrelated to why I removed this part. I remove it because it does nothing. Whether `isError` is true or false the function also just return

Reply:
Ohh that's true; didn't realise 😅 It's fine then.